operator/deploy/deploy-kiali-operator.sh

```diff
@@ -643,14 +643,16 @@ fi
 
 # If asking for the last release of Kiali (which is the default), then pick up the latest release.
 # Note that you could ask for "latest" - that would pick up the current image built from master.
```

Comment:
This is confusing.  I think we should deprecate the "latest" and "lastrelease" versions in favor of "master" and "release".  And keep "dev".

Reply:
"latest" is the common industry term that everyone knows about (think docker and the semantics about the "latest" image tag - that is the "latest" this is talking about).

"lastrelease" is simply what it refers to - the "last release" of the Kiali image.

Reply:
It's not the last release.  It's the most recent release.  Last means last, as in no more to follow.  Anyway, the industry is confused ;)